next.config.js

```diff
@@ -32,7 +32,13 @@ module.exports = nextTranslate({
     const basemapUrl = getUrlWithoutPath(
       process.env.NEXT_PUBLIC_MAPTILER_BASEMAP_URL
     )
-    const sdkUrl = getUrlWithoutPath(process.env.NEXT_PUBLIC_SUPABASE_SDK_URL)
+    /**
+     * This might not be needed
```

Comment:
We might want to verify this and remove this comment, which doesn't really help and clutters the code.

Reply:
Removed. since I guess we need this URL to allow requests from the frontend to the postgrest api